lib/metadata/bucketfile/backend.js

```diff
@@ -0,0 +1,406 @@
+import net from 'net';
+import fs from 'fs';
+import cluster from 'cluster';
+import events from 'events';
+import assert from 'assert';
+
+import level from 'level';
+import multilevel from 'multilevel';
+import sublevel from 'level-sublevel';
+import arsenal from 'arsenal';
+
+import { errors } from 'arsenal';
+import BucketInfo from '../BucketInfo';
+import constants from '../../../constants';
+
+const METADATA_PORT = 9990;
+const METADATA_PATH = '/metadata/';
+const MANIFEST_JSON = 'manifest.json';
+const MANIFEST_JSON_TMP = 'manifest.json.tmp';
+const ROOT_DB = 'rootDB';
+const METASTORE = '__metastore';
+const OPTIONS = { sync: true };
+
+let rootDB;
```

Comment:
These two vars can be moved to `startServer()` method as they are not being used anywhere else.